sequencer/src/block2.rs

```diff
@@ -98,38 +261,88 @@ mod test {
             vec![],
         ];
 
-        for tx_payloads in test_cases {
+        let vid = test_vid_factory();
+        let num_test_cases = test_cases.len();
+        for (t, tx_bodies) in test_cases.into_iter().enumerate() {
+            println!(
```

Comment:
Better to use `tracing::info`. You can call `async_compatibility_layer::logging::{setup_logging, setup_backtrace}` at the beginning of the test to set this up to output nicely to the terminal